### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by MarkStarikov**

### CODE REVIEW

**Comment by MarkStarikov** [comment]
Homework 4 Polimorfism in java

File: Homework4_1/src/Main.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+public class Main {
+
+        public static void main(String[] args) {
+
+            Areable[] array = new Areable[3];
```
Comment (shaitanaleksander) [inline Homework4_1/src/Main.java:5]: array is a bad name

File: Homework4_1/src/Main.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            array [0] = new Circle();
+            array [1] = new Rectangle();
+            array [2] = new Square();
+            System.out.println("Sum of the all massive`s geometric figures is "+Main.calcSum(array)+ " sm");
+
+        }
+
+    private static double calcSum(Areable[] array) {
+
+          double sum = array [0].calcAreaFigure() + array [1].calcAreaFigure() +array [2].calcAreaFigure(); //А что, так можно было????
```
Comment (shaitanaleksander) [inline Homework4_1/src/Main.java:15]: better to iterate using for or while, comments need to be in English

File: Homework4_2/src/Main.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                System.out.println("Participant " + participant.getName() + "  passed the obstacle " + obstacle.getName() + " at the distance " + obstacle.chekingObstacle(participant)+ " m");
+
+                /*if (!obstacle.toOvercome(participant)) {
+                    System.out.println("Participant " + participant + " did not pass the obstacle " + obstacle + " at the high " + obstacle.getHight() + ". Passed " + obstacle.getLength());
+                    break;
+                }
+                else {
+                    System.out.println("Participant " + participant + "  passed the obstacle " + obstacle + " at the high " + obstacle.getHight());
+                } */
+
```
Comment (shaitanaleksander) [inline Homework4_2/src/Main.java:37]: commented code need to be removed

**Pull request merged**